main_OOP_task_company_class.py

```diff
@@ -0,0 +1,85 @@
+# Create a class describing any company. For example, Toshiba or Apple
+
+class Toshiba:
+
+    def __init__(self, company_name: str = None, founded_year: int = None, founder_name: str = None,
+                 employees: int = None, revenue: int = None):
+        self.__company_name = company_name
+        self.__founded_year = founded_year
+        self.__founder_name = founder_name
+        self.__employees = employees
+        self.__revenue = revenue
+
+    @property
+    def company_name(self):
+        if isinstance(self.__company_name, str) and len(self.__company_name) <= 50:
+            return self.__company_name
+        else:
+            raise TypeError('Company name must be a string with no more than 50 characters.')
```

Comment:
it is a getter. So you do not need any additional logic here actually 

Please fix in all places